install_el_apache.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+sudo apt-get update
+sudo apt-get install -y apache2
+sudo systemctl start apache2
+sudo systemctl enable apache2
```

Comment:
Most likely it will fail. You don't need sudo. In the case of userdata, all commands are run as sudo by default.